v3/src/components/graph/components/dotplotdots.tsx

```diff
@@ -201,7 +201,7 @@ export const DotPlotDots = memo(observer(function DotPlotDots(props: IProps) {
       dataConfiguration?.getLegendColorForCase])
 
   usePlotResponders({
-    dataset, xAxisModel, yAxisModel, primaryAttrID, secondaryAttrID, legendAttrID, layout,
+    graphModel, primaryAttrID, secondaryAttrID, legendAttrID, layout, dotsRef,
```

Comment:
Remove `xAxisModel` and `yAxisModel` from props above now that they're no longer used.